hardware/tests/README.md

```diff
@@ -0,0 +1,27 @@
+# Hardware tests
+
+## AHRS test
+The AHRS hardware test tests the intertial measurement unit's euler angles, 
+quaternion angles and angular velocities. Copy and paste the `hw_test_ahrs.cpp` 
+code into the `main.cpp`. Follow the instructions provided in the code to complete the hardware test.
+
+## Motor test
+The motor test spins has two modes the first of which runs the motors up to speed for 1 second
```

Comment:
The motor test has two modes, the first of which runs the motors up to speed for 1 second